app/views/submissions/show.html.erb

```diff
@@ -31,8 +31,14 @@
     <% end %>
   <% end %>
 </div>
-<%= form_for(new_comment, url: submission_create_comment_path(post.id), method: :post) do |f| %>
-  <%= f.text_area :text, cols: 60 %>
+<%= form_for(new_comment, url: submissions_path, method: :post) do |comment| %>
```

Comment:
the reason you see variables like `f` used here is because the underlying variable is an instance of a `FormBuilder`. Using `comment` here might confuse future readers and lead them to thinking it's an instance of a `Comment` class. no action here unless you absolutely want to; I just wanted to throw that out there.

Reply:
gotcha - that makes sense - I like using `f`